tobac/utils/bulk_statistics.py

```diff
@@ -0,0 +1,215 @@
+"""
+Support functions to compute bulk statistics of features, either as a postprocessing step
+or within feature detection or segmentation. 
+
+"""
+
```

Comment:
Quick fix: missing `logging` import
```suggestion
import logging
```